### PR TITLE
Updated screens, incl. OSK; major editing on text.

### PR DESCRIPTION
**Pull request by Gemba**
I gave the wireless page some attention. It got the impression it got detoriated heavily over time.

- Updated screens, including the handy on-screen-keyboard.
- Removed WEP as configuration option due to broken security.
- Added a note on open network usage.
- Marked the wifikey file section as  deprecated in favor to wpa_supplicant file in `/boot/`.
- Consitent usage of WiFi (instead of Wifi or wifi).

The image binaries may be moved around to the cloud storage.

### CODE REVIEW

**Review by cmitu — COMMENTED** [review]
Thanks for the changes, they are a welcomed update. I've left some notes for a few changes.

Not sure we should remove the WEP option until we remove it also from the `wifi` module in RetroPie-Setup, but I'm fine either way - I don't think WEP is even a realistic option for with recent Linux/Debian/etc. distros.

**Comment by Gemba** [comment]
> Not sure we should remove the WEP option until we remove it also from the `wifi` module in RetroPie-Setup, but I'm fine either way - I don't think WEP is even a realistic option for with recent Linux/Debian/etc. distros.

I removed it because the Debian Wiki for wireless has it also not longer covered: https://wiki.debian.org/WiFi/HowToUse

File: docs/Wifi.md [hunk 1]
```diff
@@ -1,62 +1,49 @@
...
 
-*NOTE:* In order to use the WiFi on the Raspberry Pi, you will need to first configure the **WLAN Country** via `raspi-config`. It’s under menu _5 Localisation options_ in `raspi-config`. You can start `raspi-config` from the `RetroPie` menu in EmulationStation or from the command line with `sudo raspi-config`.
+**Notes**:
 
-There are 5 main methods to configure Wifi:
+- In order to use the WiFi on the Raspberry Pi, you will need to first configure the **WLAN Country** via `raspi-config`. It is under menu _5 Localisation options_ -> _L4 WLAN County_ in `raspi-config` dialogs. You can start `raspi-config` from the `RetroPie` menu in EmulationStation or from the command line with `sudo raspi-config`. This step is mandatory for all solution options described below.
+- WiFi will not start up if you have an hard wired ethernet connection. After disconnecting the ethernet cable you will need to reboot to get WiFi started.
+- You may also check [SSH configuration](SSH.md#enable-ssh) on how to enable SSH on your installation.
```
[the remark this answers is not in the document]
Comment (Gemba) [reply docs/Wifi.md]: Change it. Thanks.

File: docs/Wifi.md [hunk 2]
```diff
@@ -69,11 +56,23 @@ network={
 # RETROPIE CONFIG END
 ```
 
-Make sure to include the ```RETROPIE CONFIG ...``` lines as shown to ensure that the RetroPie-Setup wifi configuration module will be able to cleanly edit/delete your configuration if you wish to change it later.
+Make sure to include the `# RETROPIE CONFIG START/END` lines as shown to ensure that the RetroPie-Setup WiFi configuration module will be able to cleanly edit/delete your configuration, if you wish to change it later.
+
+## Connecting to WiFi Without a Keyboard (Deprecated)
+
+Starting with Raspberry Pi OS Stretch, loading the wifikey file from the setup script is not necessary: Thus this section is informational only.
```
[the remark this answers is not in the document]
Comment (Gemba) [reply docs/Wifi.md]: Adapted wording.

File: docs/Wifi.md [hunk 3]
```diff
@@ -1,62 +1,49 @@
 # Configuring WiFi
 
-If you have a Raspberry Pi 3, WiFi is built into the Pi, if you have a Pi2 or earlier model, then you'll need a wifi dongle. You can check to see if your wifi dongle is compatible [here](http://elinux.org/RPi_USB_Wi-Fi_Adapters). You may want to use an USB WiFi dongle, then see [here](#using-an-external-dongle).
+If you have a Raspberry Pi 3 or later model, WiFi is built into the Pi, if you have a Pi 2 or earlier model, then you will need a WiFi dongle. You can check to see if your WiFi dongle is compatible [here](http://elinux.org/RPi_USB_Wi-Fi_Adapters). You may want to use an USB WiFi dongle, then see section [using an external dongle](#using-an-external-dongle).
```
[the remark this answers is not in the document]
Comment (Gemba) [reply docs/Wifi.md]: Reverted back to previous link text.

**Comment by cmitu** [comment]
Thank you for the changes, merging.

**Pull request merged**